docs/sdk/analytics/fragments/ios/getting-started.md

```diff
@@ -12,8 +12,8 @@ Amazon Pinpoint is a fully managed AWS service that you can use to engage with y
 
 The Amplify CLI helps setup and configure Pinpoint within your application and connect with the AWS Mobile SDK.
 
-<b>Prerequisite:</b> [Install and configure the Amplify CLI](..)<br>
-<b>Recommendation:</b> [Complete the Getting Started guide](./start)
+ - **Prerequisite:** [Install and configure the Amplify CLI](/cli/start/install)
```

Comment:
Only installing the CLI is a Prerequisite. The getting started guides per category should be focused solely on that (similar to the Auth one [here](https://docs.amplify.aws/lib/auth/getting-started?platform=js). Could you change the section to the blockquote markdown format:

```
> Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)
```